jekyll/_data/sidenav.yml

```diff
@@ -209,6 +209,8 @@ en:
       children:
         - name: Status Overview
           link: 2.0/status/
+          - name: Viewing Pipelines
```

Comment:
This is failing because it needs to be de-dented to align with the above and below keys to make yaml happy 😭

Reply:
Thanks for the help @teesloane. Fixed.